src/storage/redis_db.cc

```diff
@@ -226,6 +226,30 @@ rocksdb::Status Database::TTL(const Slice &user_key, int64_t *ttl) {
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status Database::GetExpireTime(const Slice &user_key, uint64_t *timestamp) {
```

Comment:
We can use `GetMetadata` directly instead of decoding here.

Reply:
IMHO it is hard to use since we need to pass RedisType.

Reply:
Yes, maybe we need to pass empty redis type array to ignore the type check later.

Reply:
We can use `RedisTypes::All()` now.

Reply:
@kay011 Would you mind changing this implementation to `GetMetadata` since we can use `RedisTypes::All()` now.

Reply:
@git-hulk Of course, I'm thinking about how to use it more elegantly

Reply:
How about changing the parameter `type` of `Database::GetMetadata` to `RedisTypes` like ParseMetadata?

Reply:
@git-hulk I think that is the best solution! But the existed code will change the call parameter type. So there is a trade off.

Reply:
I think that's fine since only a few places that are using this function.

Reply:
@git-hulk Nice, I will implement it as stated above.